hoodie-client/src/main/java/com/uber/hoodie/WriteStatus.java

```diff
@@ -24,112 +24,132 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 
 /**
  * Status of a write operation.
  */
 public class WriteStatus implements Serializable {
 
-    private final HashMap<HoodieKey, Throwable> errors = new HashMap<>();
+  private final HashMap<HoodieKey, Throwable> errors = new HashMap<>();
```

Comment:
Indenting has changed here throughout.. Can you reformat this file and ensure the only diffs we see here are the two methods?  https://uber.github.io/hoodie/code_structure.html

Reply:
Sorry about that. 

Im using the same code style, and it seems like it is using +2 spaces for indentation.  https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation